cron.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Dans la doc Scalingo https://doc.scalingo.com/platform/app/task-scheduling/scalingo-scheduler#enabling-the-scalingo-scheduler la syntaxe a plutôt l'air d'être ça

```json
{
  "jobs": [
    {
      "command": "0 0 * * * php bin/console app:user_tokens:purge"
    }
  ]
}
```